Bottom: 5eca70891af237ca11a9eb60390ba003de74452b
Top:    ef0deb43f41fb0b9919937806b5480a282d6ba20
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 14:12:15 -0500



---
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
